modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -101,49 +115,49 @@ def number_and_street
     end
 
     def city
-      veteran.dig('address', 'city') || ''
+      veteran_data.dig('address', 'city') || ''
     end
 
     def state_code
-      veteran.dig('address', 'stateCode') || ''
+      veteran_data.dig('address', 'stateCode') || ''
     end
 
     def country_code
       return '' unless address_combined
 
-      veteran.dig('address', 'countryCodeISO2') || 'US'
+      veteran_data.dig('address', 'countryCodeISO2') || 'US'
     end
 
     def zip_code
       if zip_code_5 == '00000'
-        veteran.dig('address', 'internationalPostalCode') || '00000'
+        veteran_data.dig('address', 'internationalPostalCode') || '00000'
       else
         zip_code_5
       end
     end
 
     def zip_code_5
-      veteran.dig('address', 'zipCode5') || '00000'
+      veteran_data.dig('address', 'zipCode5') || '00000'
     end
 
     def veteran_phone_number
       veteran_phone.to_s
     end
 
     def veteran_phone_data
-      veteran&.dig('phone')
+      veteran_data&.dig('phone')
     end
 
     def veteran_homeless?
       form_data&.dig('data', 'attributes', 'veteran', 'homeless')
     end
 
     def email
-      veteran&.dig('emailAddressText').to_s.strip
+      veteran_data&.dig('emailAddressText').to_s.strip
     end
 
     def email_v2
```

Comment:
This will be removed in PR part II. Removing here will cause form data test to fail and then domino to our generator spec.